project/AppDependencies.scala

```diff
@@ -1,25 +1,22 @@
 import sbt._
 
 object AppDependencies {
-  import play.core.PlayVersion
 
-  val compile = Seq(
+  val compile: Seq[ModuleID] = Seq(
     play.sbt.PlayImport.ws,
-    "uk.gov.hmrc"       %% "bootstrap-frontend-play-28"     % "7.13.0",
+    "uk.gov.hmrc"       %% "bootstrap-frontend-play-28"     % "8.4.0",
```

Comment:
You could make assign the version number of bootstrap to a value and use this for both the app and test dependencies for bootstrap to ensure they are always the same version

Reply:
Done.